chain-impl-mockchain/src/key.rs

```diff
@@ -116,6 +118,12 @@ impl property::Deserialize for Hash {
 impl property::BlockId for Hash {}
 impl property::TransactionId for Hash {}
 
+impl fmt::Display for Hash {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
```

Comment:
oops: #441

Reply:
Rebased to only add the `BlockDate` impl.